Add note that CSN Interop doesn't support array of scalar types (#66)
SAP/csn-interop-specification: docs/primer.md

Note: As a general convention we recommend to omit optional boolean properties, if the value is `false`.

### Cardinality

> Please note that CSN Interop does not support arrays of scalar types, only of [cds.Association](./spec-v1/csn-interop-effective#association-type) and [cds.Composition](./spec-v1/csn-interop-effective#composition-type), which can express this in their [cardinality](./spec-v1/csn-interop-effective#cardinality-object) object.

Reviewer: this sounds like CSN interop supports "arrays of cds.Association/Composition", which is not the case

Author: yes, what I meant is that cdsAssociation has cardinality ("to many"), which is essentially an array of pointers. 

Any suggestions how to phrase it better?

```suggestion
Please note that CSN Interop does not support arrays of scalar types. There is [cds.Association](./spec-v1/csn-interop-effective#association-type) and [cds.Composition](./spec-v1/csn-interop-effective#composition-type), where (to many) [cardinality](./spec-v1/csn-interop-effective#cardinality-object) can be defined.
```

Author: @stewsk - I've updated the descriptions, is it better now?

The reason is that arrays of scalar types are not supported in all tech-stacks and would require a more complex mapping.
By not supporting this in CSN Interop, the model needs to make the decision how multi-value scalar types are to be mapped.

This can be done, e.g. by:

- converting the scalar type to an Entity and use `cds.Association` or `cds.Composition` with the appropriate cardinality (RECOMMENDED).

> - converting the array of values into a single string, e.g. by using a delimiter or parsing it to a JSON string.

Author: If this is done, it would be consequent to also have a `@Semantics` annotation to indicate this. E.g. `@Semantics.json` or `@Semantics.commaSeparated`, `@Semantics.semicolonSeparated`
